continuous_integration/travis/modules_test.sh

```diff
@@ -94,12 +94,3 @@ for f in spyderlib/*/*/*.py; do
         exit 1
     fi
 done
-
-
-# Spyderplugins
-for f in spyderplugins/widgets/*.py; do
```

Comment:
Please don't remove these tests. Just change this line to

    for f in spyplugins/ui/*/widgets/*.py; do

That should fix these tests because spyplugins widgets can be run outside Spyder.